_data/CharlesLiu02/CharlesLiu02.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Charles Liu
+institution: UC Berkeley
+profile_pic: charles.jpg
```

Comment:
Git is case-sensitive. Could you please fix the file name? You could do one of the following:
1. Change `charles.jpg` to `charles.JPG ` in the markdown file.
2. Rename `charles.JPG ` to `charles.jpg`.
I'd suggest you rename the file!